packages/frontend/core/src/components/blocksuite/block-suite-editor/lit-adaper.tsx

```diff
@@ -97,9 +97,7 @@ const usePatchSpecs = (page: Doc, shared: boolean, mode: DocMode) => {
       patchReferenceRenderer(patched, reactToLit, referenceRenderer),
       confirmModal
     );
-    if (!page.readonly) {
```

Comment:
This might lead to all peek-able blocks being peeked, which probably isn't expected. Perhaps CenterPeek should provide some capability to support configuration? @pengx17

Reply:
Currently, there are four blocks can be peekable:
-` image`
- `surface-ref`  (insert Frame or Group into page)
- `embed-linked-doc`
- `embed-syncd-doc`

The `embed-linked-doc` and `embed-syncd-doc` will show a **empty banner** in shared doc, since their render logic is override by https://github.com/toeverything/AFFiNE/blob/6c82148b5e266a0807c5e7724d84fedf0d29f74e/packages/frontend/core/src/components/blocksuite/block-suite-editor/specs/custom/spec-patchers.tsx#L514-L540

Reply:
Seems card view link can still be peeked, and show a loading page.

Reply:
However, I think it is better provide a interface to enable or disable a peekable block based on some context. Like 
```ts
@Peekable({
    showWhen: (ctx) : boolean => {/* ... */}
})
```

Reply:
> Seems card view link can still be peeked, and show a loading page.

@Catjuice I have disabled both embed cards peekable in PR https://github.com/toeverything/blocksuite/pull/7596, which has been bump to AFFiNE.